tests/test_tags.py

```diff
@@ -342,3 +342,20 @@ def test_input_boolean_attributes_with_other_attrs() -> None:
     # Test disabled with other attributes
     html = _r(air.Input(name="readonly", type="text", value="Can't change me", disabled=True, readonly=True))
     assert html == '<input name="readonly" type="text" value="Can\'t change me" readonly disabled>'
+
+
+def test_comment_tag() -> None:
```

Comment:
Can you add a test covering multi-line comments so we can confirm the correct behavior?

Reply:
Yes please!